file-ballerina/src/file/file.bal

```diff
@@ -18,37 +18,25 @@ import ballerina/java;
 
 # Returns the current working directory.
 # ```ballerina
-# string dirPath = file:getCurrentDirectory();
+# string dirPath = file:getCurrentDir();
 # ```
 # 
 # + return - Current working directory or else an empty string if the current working directory cannot be determined
-public isolated function getCurrentDirectory() returns string = @java:Method {
+public isolated function getCurrentDir() returns string = @java:Method {
     'class: "org.ballerinalang.stdlib.file.nativeimpl.Utils",
     name: "getCurrentDirectory"
 } external;
 
-# Reports whether the file or directory exists in the given the path.
-# ```ballerina
-# boolean result = file:exists("foo/bar.txt");
-# ```
-#
-# + path - String value of the file path
-# + return - True if the path is absolute or else false
-public isolated function exists(@untainted string path) returns boolean = @java:Method {
-    'class: "org.ballerinalang.stdlib.file.nativeimpl.Utils",
-    name: "exists"
-} external;
-
 # Creates a new directory with the specified file name.
 # If the `parentDirs` flag is true, it creates a directory in the specified path with any necessary parents.
 # ```ballerina
 # string | error results = file:createDir("foo/bar");
 # ```
 #
 # + dir - Directory name
-# + parentDirs - Indicates whether the `createDir` should create non-existing parent directories
+# + option - Indicates whether the `createDir` should create non-existing parent directories
```

Comment:
Shall we mention the default value. Something like `This defaults only to the given directory`